### PR TITLE
TAUR-1379 Simple utility to recreate queues from one instance in another.

### PR DESCRIPTION
**Pull request by oxtopus**
Example usage:

```
nta-import-queues --source-host=REDACTED --source-user=REDACTED --source-passwd=REDACTED --destination-host=REDACTED --destination-user=REDACTED --destination-passwd=REDACTED --queue-regex="taurus\.mswapper\.model\.input\..+"
```

### CODE REVIEW

File: nta.utils/nta/utils/tools/import_queues.py [hunk 1]
```diff
@@ -0,0 +1,134 @@
...
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+# See the GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see http://www.gnu.org/licenses.
+#
+# http://numenta.org/licenses/
+# ----------------------------------------------------------------------
+
+from optparse import OptionGroup, OptionParser
```
Comment (jcasner) [inline nta.utils/nta/utils/tools/import_queues.py]: `optparse` is deprecated as of python 2.7.  Can you please use `argparse` instead?

File: nta.utils/nta/utils/tools/import_queues.py [hunk 2]
```diff
@@ -0,0 +1,134 @@
...
+                  help=("Queues with names that match this pattern will be"
+                        " copied from source into destination.  If not"
+                        " specified, the default pattern '.*' will be used to"
+                        " copy all queues."))
+
+
+
+def _getClient(host, user, passwd, _cache={}):
+  """
+  :param str host: RabbitMQ host
```
Comment (jcasner) [inline nta.utils/nta/utils/tools/import_queues.py]: The convention I've seen used elsewhere is 
```
:param host: RabbitMQ host
:type host: str
```

Comment (oxtopus) [inline nta.utils/nta/utils/tools/import_queues.py]: Both are technically valid: http://sphinx-doc.org/domains.html#info-field-lists

Comment (jcasner) [inline nta.utils/nta/utils/tools/import_queues.py]: Cool.  We should probably put our preferred approach into our coding standards doc for clarity in the future...

Comment (vitaly-krugl) [inline nta.utils/nta/utils/tools/import_queues.py]: @jcasner: I started using the `:param type arg:` syntax a little while ago, and prefer it over the two-line approach. The two-line approach still has its place, such as when <type> description needs to consist of multiple space-separated words.

**Pull request merged**